check-pr-description/action.yaml

```diff
@@ -60,7 +80,16 @@ runs:
         pr_contains_string=$(echo $pr_text | jq 'test("${{ inputs.string }}")' || echo "false")
         echo "pr-contains-string=$pr_contains_string" >> $GITHUB_OUTPUT
         echo "String found: $pr_contains_string"
-        test_enabled=true
+        if [ "${{ inputs.test-enabled-default }}" = "true" ]; then
+          test_enabled=true
+        else
+          test_enabled=false
+        fi
+        if [ "${{ inputs.test-legacy-enabled-default }}" = "true" ]; then
+          test_legacy_enabled=true
+        else
+          test_legacy_enabled=false
+        fi
```

Comment:
It's probably nicer to use direct assignments here.

Reply:
You mean using just 

```
test_enabled="${{ inputs.test-enabled-default }}"
test_legacy_enabled="${{ inputs.test-legacy-enabled-default }}"
```
?

Reply:
If it's doable to assign directly `test_legacy_enabled=<result of comparing to "true">`, but it's not important.

Reply:
I tried a couple of solutions but I didn't manage to assign directly something like "result of" because the comparison `[ "${{ inputs.test-legacy-enabled-default }}" = "true" ]` seems to return an exit code but not something assignable.

My best attempt is the following, but it reads worse than the current if/else

```
test_legacy_enabled=$([ "${{ inputs.test-legacy-enabled-default }}" = "true" ] && echo true || echo false)
```